### PR TITLE
Updated configs to helmchart version

### PR DESCRIPTION
**Pull request by dabasov**
## Changes
Merged helm charts version
Adjusted costs to loadtest results
## Tests 
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Review by service-0chain — APPROVED** [review]

File: docker.local/config/sc.yaml [hunk 1]
```diff
@@ -199,13 +197,16 @@ smart_contracts:
...
+      blobber_usage_ratio: 40
+    expose_mpt: true
     cost:
       update_settings: 100
       read_redeem: 100
       commit_connection: 100
       new_allocation_request: 3000
       update_allocation_request: 2500
-      finalize_allocation: 100
+      finalize_allocation: 9500
```
Comment (peterlimg) [inline docker.local/config/sc.yaml:209]: Any reason to change the value?

File: docker.local/config/0chain.yaml [hunk 2]
```diff
@@ -16,26 +14,29 @@ development:
...
     miner: true
     multisig: true
     vesting: true
+    magma: false #set true to enable magma
     zcn: true
   txn_generation:
     wallets: 50
+    transactions: 70
+    pour_point: 100
+    pour_amount: 10000
```
Comment (peterlimg) [inline docker.local/config/0chain.yaml]: Are these new added configs being used? Perhaps we should do these changes along with the code that use them.

File: docker.local/config/0chain.yaml [hunk 3]
```diff
@@ -48,32 +49,33 @@ server_chain:
     max_block_size: 10
     max_block_cost: 10000 #equal to 100ms
     max_byte_size: 1638400
+    generators: 2
```
Comment (peterlimg) [inline docker.local/config/0chain.yaml]: What does the `generators` do, we are getting generators number base on the `generators_percent` and if it is less than the `min_generators`, we will use the `min_generators`.

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: docker.local/config/0chain.yaml (hunk 2 above)
Comment (dabasov) [reply docker.local/config/0chain.yaml]: it is not used, removed

File: docker.local/config/0chain.yaml (hunk 3 above)
Comment (dabasov) [reply docker.local/config/0chain.yaml]: the same

File: docker.local/config/sc.yaml [hunk 4]
```diff
@@ -218,7 +219,7 @@ smart_contracts:
       curator_transfer_allocation: 100
       challenge_request: 100
       challenge_response: 1600
-      generate_challenges: 9999 #exceeds limit, might be switched off
+      generate_challenges: 0
```
Comment (peterlimg) [inline docker.local/config/sc.yaml:222]: Generate challenge should have been optimized, so perhaps we ca use 100 for it now.

File: docker.local/config/sc.yaml (hunk 1 above)
Comment (peterlimg) [inline docker.local/config/sc.yaml:209]: Is finalize_allocation sc optimized? if so we should not use 9500 perhaps

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Pull request closed without merging**